experiments/experiments-api/src/main/java/com/duckduckgo/experiments/api/VariantManager.kt

```diff
@@ -38,4 +38,10 @@ interface VariantManager {
      * has been updated.
      */
     fun saveVariants(variants: List<VariantConfig>)
+
+    /**
+     * Updates the configDownloaded flag to indicate that the variant should be assigned,
+     * as the variantManager config is now ready
+     */
+    fun variantConfigDownloaded()
```

Comment:
Ditto on a different comment.
This API directly couple the privacy config implementation details with variant manager.
If we want to be notified when config is downloaded, we should just create a callback mechanism in privacy config (through interface plugin) that will be called ever time config is updated. No the other way around

Reply:
I think you saw this in a previous commit. I implemented this public interface first to know when the variants will be available to allocate. However, moving the download() as a public method I can make sure I have variants fetched for when atb is initialised resolved this problem.